barretenberg/cpp/src/barretenberg/relations/auxiliary_relation.hpp

```diff
@@ -97,8 +111,9 @@ template <typename FF_> class AuxiliaryRelationImpl {
     {
         BB_OP_COUNT_TIME_NAME("Auxiliary::accumulate");
         // All subrelations have the same length so we use the same length view for all calculations
-        using Accumulator = typename std::tuple_element_t<0, ContainerOverSubrelations>;
+        using Accumulator = typename std::tuple_element_t<3, ContainerOverSubrelations>;
         using View = typename Accumulator::View;
+        using ShortView = typename std::tuple_element_t<0, ContainerOverSubrelations>::View;
```

Comment:
I think it would be nice to add the related comment from the PR description here